src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Components/ApiVersionHelper.cs

```diff
@@ -108,7 +109,7 @@ private static string[] GetApiVersionsForResourceType(AzureContext context, stri
                 ? "/providers"
                 : string.Format("/subscriptions/{0}/providers", defaultSubscription.Id);
 
-            var providers = PaginatedResponseHelper.Enumerate(
+            ResourceProviderDefinition[] providers = PaginatedResponseHelper.Enumerate(
```

Comment:
Before I made this change, VS build failed due to ambiguous type and then I have to make this change.